cmd/createorupdate/createorupdate.go

```diff
@@ -1,8 +1,11 @@
 package main
 
 import (
+	"bytes"
 	"context"
+	"crypto/hmac"
```

Comment:
is doing this needed if we're listening on localhost?

Reply:
It depends on how we end up running this in CI. I am thinking that separating the client and the server into separate processes probably makes sense. I still don't know whether that is going to be two separate pods running or two containers in a pod would suffice and we can still use localhost.

Reply:
Going to remove until we actually use it.